src/basilisp/process.lpy

```diff
@@ -276,7 +276,9 @@
   (let [process (apply start opts+args)
         retcode (.wait process)]
     (if (zero? retcode)
-      (slurp (.-stdout process))
+      (if-let [out (.-stdout process)]
+        (slurp out)
```

Comment:
This just works differently than in Java. Python doesn't attach any `stdout` to the process object if you inherit the current process stdout, so we just have to check and return an empty string in that case.